lib/index.js

```diff
@@ -33,6 +34,25 @@ NHSPrototypeKit.init = function (options) {
   // Add the prototype kit Nunjucks filters
   NHSPrototypeKit.nunjucksFilters.addAll(options.nunjucks)
 
+  // Use assets from NHS frontend
+  options.express.use(
+    '/nhsuk-frontend',
+    express.static(
+      join(import.meta.dirname, '../node_modules/nhsuk-frontend/dist/nhsuk')
+    )
+  )
+
+  // Use assets from lib
+  options.express.use(
+    '/nhsuk-prototype-kit-assets',
+    express.static(join(import.meta.dirname, '../lib/assets'))
```

Comment:
If you use `resolve()` instead of `join()` it'll fully the path without `../`

Thankfully, both will flip the slash direction for Windows though

Reply:
Any advantage to using `resolve()` here? It's a relative path to the current folder (as it's within the package)?

Reply:
Pushed, try now?